assets/css/main.css

```diff
@@ -4910,6 +4910,11 @@ input.field--reset:active {
       color: #00a2c7 !important;
       font-weight: 600; } }
 
+@media only screen and (min-width: 640px) {
+  .navItemLink--experiences .navItem-label {
+    color: #5c5cff !important;
```

Comment:
Do we specifically want this to be a color not in swarm?

Reply:
Hi @hsbacot, we got the 'okay' from design to proceed with using this distinctive purple color. Right now this is the only place it's used outside the experiences repos. If we need to use it in a second place, I'll work on getting it added to the official color constants in Swarm. That okay?

Reply:
Out of curiosity, does swarm already have a purple color? And if so, why are we designing outside of swarm colors?

Reply:
@sadafie do I have access to swarm-ui colors in `meetup-web-components`? I just tried to use `$C_veridian` and got an undefined variable error.

Reply:
@carawarner It would be $color_viridian if importing the scss variables from `@meetup/swarm-constants` or `--color-viridian` if using the css variables (recommended)